tooling/noir_js_backend_barretenberg/src/index.ts

```diff
@@ -20,7 +20,7 @@ export class BarretenbergBackend implements Backend {
   private acirUncompressedBytecode: Uint8Array;
 
   constructor(
-    private acirCircuit: CompiledCircuit,
+    acirCircuit: CompiledCircuit,
```

Comment:
The changes to comments all make sense wrt the PR, but I can't speak for the dropping of `private` here.

Reply:
Having `private` marks that this argument should be turned into a field on the class like `acirUncompressedCircuit`. We don't actually use `acirCircuit` outside of the constructor so we don't need to keep it around afterwards.

Reply:
Ah okay. In general a `private` flag being dropped (making it public), I felt was worth double checking. Cheers.